test/bigtest/tests/suppress-from-discovery-tests.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { Form } from 'react-final-form';
+
+import { mountWithContext } from '../helpers/mountWithContext';
+
+import { SuppressFromDiscoveryFieldArray } from '../../../src/settings/components';
+import SuppressFromDiscoveryInteractor from '../interactors/suppress-from-discovery';
+
+const interactor = new SuppressFromDiscoveryInteractor();
+let formValues = {};
+
+describe('Suppress from discovery tests', () => {
+  describe('Rendering the "Suppress from discovery" settings component', () => {
+    beforeEach(async function () {
+      await mountWithContext(
+        <Form
+          onSubmit={() => {}}
+          render={({ handleSubmit, values }) => {
+            formValues = values;
+            return (
+              <form onSubmit={handleSubmit}>
+                <SuppressFromDiscoveryFieldArray name="test" />
+              </form>
+            );
+          }}
+          subscription={{ values: true }}
+        />
+      );
+    });
+
+    it('renders the description', () => {
+      expect(interactor.isDescription).to.be.true;
+    });
+
+    describe('agreement line checkbox tests', () => {
+      it('renders the agreement line checkbox', () => {
+        expect(interactor.isAgreementLine).to.be.true;
```

Comment:
The naming here needs to be more specific for `isAgreementLine`, maybe `isAgreementLineCheckBox` and similaly for the pci as well